pkg/cmd/server/origin/legacy.go

```diff
@@ -207,6 +207,8 @@ func LegacyStorage(storage map[schema.GroupVersion]map[string]rest.Storage) map[
 
 				case *imagestreametcd.REST:
 					legacyStorage[resource] = &imagestreametcd.LegacyREST{REST: storage}
+				case *imagestreametcd.LayersREST:
+					delete(legacyStorage, resource)
```

Comment:
This shouldn't be necessary.  Don't add yourself to the whitelist above and you should be good.

Reply:
> This shouldn't be necessary. Don't add yourself to the whitelist above and you should be good.

this is required for some reason?